sar-qsar-environ-res.csl

```diff
@@ -0,0 +1,164 @@
+      <label form="short" text-case="lowercase" prefix=", "/>
+    </names>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="[" suffix="]" delimiter=",">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" second-field-align="flush" et-al-min="7" et-al-use-first="6">
+    <layout suffix=".">
+      <text variable="citation-number" prefix="[" suffix="]"/>
+      <choose>
+        <if type="article-journal" match="any">
+          <text macro="author"/>
+          <text variable="title" font-style="italic" suffix=", "/>
+          <text variable="container-title" form="short" suffix=" "/>
+          <number suffix=" " variable="volume"/>
+          <date date-parts="year" form="text" variable="issued" prefix="(" suffix="), "/>
+          <text variable="page" prefix="pp. "/>
+        </if>
+        <else-if type="book" match="any">
+          <text macro="author"/>
+          <text variable="title" font-style="italic" suffix=", "/>
+          <choose>
+            <if match="any" variable="edition">
+              <text variable="edition" suffix="ed., "/>
+            </if>
+          </choose>
+          <choose>
+            <if match="any" variable="volume">
+              <text variable="volume" prefix="Vol. " suffix=", "/>
+            </if>
+          </choose>
+          <choose>
+            <if match="any" variable="collection-title">
+              <text variable="collection-title"/>
+              <number prefix=" Vol. " suffix="," variable="collection-number"/>
+            </if>
+          </choose>
+          <text variable="publisher" suffix=", "/>
+          <text variable="publisher-place" suffix=", "/>
+          <date date-parts="year" form="text" variable="issued"/>
+        </else-if>
+        <else-if type="chapter" match="any">
+          <text macro="author"/>
+          <text variable="title" font-style="italic" suffix=", "/>
+          <text variable="container-title" font-style="italic" prefix="in " suffix=", "/>
+          <text macro="editor" suffix=", "/>
+          <text variable="publisher" suffix=", "/>
+          <text variable="publisher-place" suffix=", "/>
+          <date date-parts="year" form="text" variable="issued" suffix=", "/>
+          <text variable="page" prefix="pp. "/>
+        </else-if>
+        <else-if type="thesis" match="any">
+          <text macro="author"/>
+          <text variable="title" font-style="italic" suffix=", "/>
+          <text variable="genre" suffix=", "/>
+          <text variable="publisher" suffix=", "/>
+          <date date-parts="year" form="text" variable="issued"/>
+        </else-if>
+        <else-if type="webpage" match="any">
+          <text variable="publisher" suffix=", "/>
+          <text variable="title" font-style="italic"/>
+          <text variable="URL" prefix="; available at "/>
```

Comment:
with the exception of software, I'm seeing ". Available at" rather than "; available at"